packages/CopyInput/stories/variations/Variations.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Gap } from "storybook/assets/styles/common.styles";
+import Input from "@paprika/input";
+import CopyInput from "../../src/CopyInput";
+
+const CopyInputVariations = () => {
+  const customInputValue = "this is my custom Input component";
+
+  return (
+    <>
+      <CopyInput value="read only" />
+      <Gap.Small />
+      <CopyInput isReadOnly={false} value="edit input" />
+      <Gap.Small />
+      <CopyInput isReadOnly={false} value="an error occurred">
+        <CopyInput.Input hasError />
+      </CopyInput>
+      <Gap.Small />
+      <CopyInput hasValueContainer value="button kind=minor and hasValueContainer prop used">
+        <CopyInput.Button kind="minor" />
+      </CopyInput>
+      <Gap.Small />
+      <CopyInput isReadOnly={false} value="button kind=default">
+        <CopyInput.Input hasClearButton />
+        <CopyInput.Button kind="default" />
+      </CopyInput>
+      <Gap.Small />
+      <>
+        <Input hasClearButton size={Input.types.size.LARGE} value={customInputValue} />
+        <CopyInput hasInputContainer={false} value={customInputValue} />
+      </>
+    </>
```

Comment:
One additional use case I'd suggest illustrating:

```jsx
<CopyInput value="something" hasInputContainer={false} />
```

This will just render a button that copies the `value` to the clipboard.